web/docs/data-model/prisma-file.md

```diff
@@ -0,0 +1,200 @@
+
+Wasp uses the `datasource` block to know which database you want to use (PostgreSQL in this case) and some other options.
+
+Wasp uses the `generator` block to generate the Prisma Client code that you can use in your application to interact with the database.
+
+Finally, Prisma models represent Wasp Entities which can be then used in the `main.wasp` file:
+
+```wasp title="main.wasp"
+app myApp {
+  wasp: {
+    version: "^0.14.0"
+  },
+  title: "My App",
+}
+
+...
+
+// Using Wasp Entities in the Wasp file
+
+query getTasks {
+  fn: import { getTasks } from "@src/queries",
+  // highlight-next-line
+  entities: [Task]
+}
+
+job myJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@src/workers/bar"
+  },
+  // highlight-next-line
+  entities: [Task],
+}
+
+api fooBar {
+  fn: import { fooBar } from "@src/apis",
+  // highlight-next-line
+  entities: [Task],
+  httpRoute: (GET, "/foo/bar/:email")
+}
+
+```
+
+In the implementation of the `getTasks` query, `Task` is a Wasp Entity that corresponds to the `Task` model defined in the `schema.prisma` file. 
+
+The same goes for the `myJob` job and `fooBar` API, where `Task` is used as an Entity.
+
+## Wasp-specific Prisma configuration
+
+Wasp mostly lets you use the Prisma schema file as you would in any other JS/TS project. However, there are some Wasp-specific rules you need to follow.
+
+### The `datasource` block
+
+```prisma title="schema.prisma"
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Wasp will take the `datasource` you write and use it as-is.
+
+There are some rules you need to follow:
+- You can only use `"postgresql"` or `"sqlite"` as the `provider` because Wasp only supports PostgreSQL and SQLite databases. 
+- You must set the `url` field to `env("DATABASE_URL")` so that Wasp can work properly with your database.
+
+### The `generator` blocks
+
+```prisma title="schema.prisma"
+generator client {
+  provider = "prisma-client-js"
+}
+```
+
+Wasp requires that there is a `generator` block with `provider = "prisma-client-js"` in the `schema.prisma` file.
+
+You can add additional generators if you want to use them in your project.
+
+### The `model` blocks
+
+```prisma title="schema.prisma"
+model User {
+  id      Int        @id @default(autoincrement())
+  tasks   Task[]
+}
+
+model Task {
+  id          Int        @id @default(autoincrement())
+  description String
+  isDone      Boolean    @default(false)
+  user        User       @relation(fields: [userId], references: [id])
+  userId      Int
+}
+```
+
+You can define your models in any way you like, if it's valid Prisma schema code, it will work with Wasp.
+
+Some minor caveats to keep in mind:
+
+- Wasp doesn't support `/// comment` syntax in the `schema.prisma` file. We are tracking it [here](https://github.com/wasp-lang/wasp/issues/2132), let us know if this is something you need.
```

Comment:
```suggestion
- Wasp doesn't fully support `/// comment` syntax in the `schema.prisma` file yet. We are tracking it [here](https://github.com/wasp-lang/wasp/issues/2132), let us know if this is something you need.
```
I added "yet" because you can write those comments if I am correct, so we will not crash on them, but yeah, we will not forward them to genreated code.